controller/monitor.go

```diff
@@ -205,7 +205,7 @@ func (m *MonitorMgr) Remove(appName string) {
 			if len(parts) != 2 {
```

Comment:
i believe this check here will bypass the code below since now there is one extra part so this needs to change to 
`if len(parts) != 3`